docs/reference/modules/_template.md

```diff
@@ -1,9 +1,15 @@
+---
+extends: '@apostrophecms/module'
+---
+
 # `@apostrophecms/module-name`
 
 **Alias:** `apos.i18n`
 
 *General description paragraph.* Cras mattis consectetur purus sit amet fermentum. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
 
+**Extends:** `{{ $frontmatter.extends }}`
```

Comment:
I don't really think this will be a problem for that. Most devs won't be in these docs often and code examples will guide them the correct way. We use "extends" elsewhere:
> Inheritance is the glue of the module system. Every module extends another module, inheriting functionality and structure. This means that your blog post module, which extends the ["piece type"](/reference/glossary.md#piece) module, comes with a huge set of features you never have to write.

If you think it's a problem though, I can use "inherits from," like A2, though it feels clunkier to me.